client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -60,6 +95,84 @@ const StatsCommercialPurchase = ( {
 	);
 };
 
+const StatsPersonalPurchase = ( {
+	siteSlug,
+	maxSliderPrice,
+	pwywProduct,
+	redirectUri,
+	from,
+	adminUrl,
+	disableFreeProduct = false,
+}: StatsPersonalPurchaseProps ) => {
+	const translate = useTranslate();
+	const sliderStepPrice = pwywProduct.cost / MIN_STEP_SPLITS;
```

Comment:
We could take a look later whether we could reduce some of the code duplication here. Definitely not a blocker. 😃